LoTAS-Fabric/versions2/fabric/Run_Fabric.launch

```diff
@@ -13,6 +14,6 @@
     <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="net.fabricmc.devlaunchinjector.Main"/>
     <stringAttribute key="org.eclipse.jdt.launching.MODULE_NAME" value="fabric"/>
     <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="fabric"/>
-    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Dfabric.dli.config=&quot;${workspace_loc:LoTAS-Fabric}/versions2/fabric/.gradle/loom-cache/launch.cfg&quot; -Dfabric.dli.env=client -Dfabric.dli.main=net.fabricmc.loader.launch.knot.KnotClient"/>
+    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Dfabric.dli.config=&quot;${workspace_loc:LoTAS-Fabric}/versions2/fabric/.gradle/loom-cache/launch.cfg&quot; -Dfabric.dli.env=client -Dfabric.dli.main=net.fabricmc.loader.launch.knot.KnotClient -XX:MaxJavaStackTraceDepth=5"/>
```

Comment:
Do you want to keep the stack trace depth limit? @MCPfannkuchenYT

Reply:
I set it at 5, i think that's a good number.

Reply:
Don't mind me if I ever change it back